tdrs-frontend/.env.development

```diff
@@ -1,7 +1,7 @@
 REACT_APP_BACKEND_URL=http://localhost:8080/v1
-REACT_APP_TIMEOUT_TIME=6000
-REACT_APP_LOGOUT_TIME=10000
-REACT_APP_DEBOUNCE_TIME=1000
+REACT_APP_TIMEOUT_TIME=1000 * 60 * 20
+REACT_APP_LOGOUT_TIME=1000 * 60 * 3
```

Comment:
Once the modal pops up, the user has three minutes to make a selection before being automatically logged out.